### ISSUE TITLE
Provide a short hand way to define every/everySuspend with all arg matchers set to any()

### ISSUE THREAD

**Issue by dalewking** [lupuuss/Mokkery#15]
In most of my use of Mocking I set up mocks without checking parameters of the call in the setup and instead check parameters in verification. It would be nice to have a way to define that without having to specify all the `any()` parameters. The way that could be done is with method references

So from the examples:

```kotlin
val repository = mock<BookRepository> {
   everySuspend { findById(any()) } calls { (id: String) -> Book(id) }
}
```

Would be able to be replaced by:

```kotlin
val repository = mock<BookRepository> {
   everySuspend(this::findById) calls { (id: String) -> Book(id) }
}
```

**Comment by lupuuss** [comment]
Good idea! It's not very straightforward to implement, but I will try it in the future.

**Comment by dalewking** [comment]
Should figure out similar idea for verification, though will have to be different
